packages/core/src/common/command.ts

```diff
@@ -48,6 +48,10 @@ export interface CommandHandler {
      * Test whether menu items for this handler should be visible.
      */
     isVisible?(...args: any[]): boolean;
+    /**
+     * Test whether menu items for this handler should be toggled.
+     */
+    isToggled?(...args: any[]): boolean;
```

Comment:
Is there a way without changing command apis?

Reply:
`isToggled` is optional. which issues do you see with that?

Reply:
@akosyakov if the command handler computes that state, it's necessary to extend the api, right. 

if we want to to have the state at the menu item, we could pass it to handler's execute. but then, it's hard to have multiple bindings.

Reply:
I guess no, there is no handler for menu item label and `isVisible` handler doesn't work for main menu items.

Reply:
ok